dulwich/tests/test_porcelain.py

```diff
@@ -57,7 +57,8 @@ class PorcelainTestCase(TestCase):
 
     def setUp(self):
         super(PorcelainTestCase, self).setUp()
-        self.test_dir = tempfile.mkdtemp()
+        self.test_dir = "/Users/romain/tests_dulwich"
```

Comment:
This is what's breaking the tests on travis. It doesn't have a directory called /Users/romain ;)

Reply:
aha my bad, i got my `tmp` dir symlinked to `/private/tmp` so I switched to this, or tests failed :p but wan't talking about that, before there was a bug were python2.7 and pypy failed but other succeeded

Reply:
@jelmer  like this one for the other PR: https://travis-ci.org/dulwich/dulwich/builds/396828073?utm_source=github_status&utm_medium=notification